core/src/main/kotlin/org/evomaster/core/output/naming/LanguageConventionFormatter.kt

```diff
@@ -0,0 +1,35 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.output.OutputFormat
+
+class LanguageConventionFormatter(
+    private val outputFormat: OutputFormat
+) {
+
+    fun formatName(testKeywords: List<String>): String {
+        return when {
+            outputFormat.isJavaOrKotlin() -> formatCamelCase(testKeywords)
+            outputFormat.isPython() -> formatSnakeCase(testKeywords)
+            outputFormat.isJavaScript() -> formatPascalCase(testKeywords)
+            else -> formatCamelCase(testKeywords)
+        }
+
+    }
+
+    private fun formatCamelCase(testKeywords: List<String>): String {
+        return testKeywords.joinToString("") { capitalize(it) }.decapitalize()
+    }
+
+    private fun formatSnakeCase(testKeywords: List<String>): String {
+        return testKeywords.joinToString("_") { it }
+    }
+
+    private fun formatPascalCase(testKeywords: List<String>): String {
+        return testKeywords.joinToString("") { capitalize(it) }
+    }
+
+    fun capitalize(word: String): String {
```

Comment:
this is already in `org.evomaster.core.utils.StringUtils`.

Reply:
Hadn't seen it, changed it